lib/Services/cayenne.js

```diff
@@ -992,6 +992,14 @@ class Cayenne extends Request {
     return this.send('GET', path);
   }
 
+  /**
+   * @param {Object} payload
+   */
+  migrateApplicationUser(payload) {
+    const path = `/move/users`;
+    const opt = { payload, response: 600000, deadline: 900000 };
```

Comment:
Can this instead be provided as a parameter?

```js
migrateApplicationUser(payload, opts) {
  opts = { response: 60000, deadline: 90000, ...opts);
  opts.payload = payload;
...
}
```

Would allow us to override the default values if we ever need to increase timeouts.

Reply:
Method changed. 👍